src/code/z_sub_s.c

```diff
@@ -248,7 +250,9 @@ s32 SubS_ChangeAnimationByInfoS(SkelAnime* skelAnime, AnimationInfoS* animations
 
 #pragma GLOBAL_ASM("asm/non_matchings/code/z_sub_s/func_8013D2E0.s")
 
-#pragma GLOBAL_ASM("asm/non_matchings/code/z_sub_s/func_8013D5E8.s")
+s32 SubS_IsAngleDiffLessThanOrEqualToThreshold(s16 angleA, s16 threshold, s16 angleB) {
+    return (ABS_ALT(BINANG_SUB(angleB, angleA)) <= threshold) ? true : false;
+}
```

Comment:
The  `? true : false` part is so unnecessary lol

I think it is common to say "LessEqual" when referring to "LessThanOrEqual". What do you think?

Reply:
That does make sense. So decided to go with `SubS_AngleDiffLessEqual`.
Also on second thought, I wonder if this function was the origin of the name `SubS` and this file was originally going to be about angle (s16) math, but eventually got bloated into what it is now.

Reply:
What about `IsAngleWithinRange`?

Reply:
I don't really like the term range, because you don't really specify a range. You only provide a main threshold value.